dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -60,7 +60,8 @@
     .mainblock
       %form#voteform{onsubmit: 'return false;'}
         - data['answers'].each_with_index do |answer, i|
-          - correct = answer['correct']
+          -# When LevelGroup-hosted, with students (or teachers doing PD), set all answers false before rendering page.
+          - correct = (standalone || current_user.try(:should_see_inline_answer?, @script_level)) ? answer['correct'] : false
```

Comment:
I wonder if this would be easier to parse if we create a helper method, i.e. something like `force_answers_false` (or better name). Then here we could do `correct = force_answers_false ? false : answers['correct']` here